ISAACConfig.cmake

```diff
@@ -0,0 +1,142 @@
+# - Config file for the isaac package
+# It defines the following variables
+#  ISAAC_INCLUDE_DIRS - include directories for FooBar
+#  ISAAC_LIBRARIES    - libraries to link against
+#  ISAAC_DEFINITIONS  - necessary definitions
+#
+# It defines the following options
+#  ISAAC_THREADING
+#  ISAAC_SHOWBORDER
+#  ISAAC_CUDA
+#  ISAAC_ALPAKA
+#  ISAAC_JPEG
+
+###############################################################################
+# ISAAC
+###############################################################################
+cmake_minimum_required (VERSION 2.6)
```

Comment:
That is a unrealistic low cmake requirement.

For example, the used `Boost 1.56.0` version was added in [~CMake 3.1.0+](https://github.com/Kitware/CMake/blob/v3.1.0/Modules/FindBoost.cmake)

Otherwise you will need `SET(Boost_ADDITIONAL_VERSIONS "1.56" "1.56.0")` [see here](https://github.com/Kitware/CMake/blob/v2.8.0/Modules/FindBoost.cmake#L73)